.github/ISSUE_TEMPLATE.md

```diff
@@ -8,11 +8,18 @@
 
 [this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]
 
+**Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before.
+Please check first before submitting so that we do not have to handle and close duplicates!
+
+**Note**: Many problems can be resolved by simply upgrading `pandas` to the latest version. Before submitting, please check
+if that solution works for you. If possible, you may want to check if `master` addresses this issue, but that is not necessary.
+
 #### Expected Output
 
 #### Output of ``pd.show_versions()``
 
 <details>
-# Paste the output here pd.show_versions() here
+
+[paste the output of ``pd.show_versions()`` here below this line]
```

Comment:
A lot of people were just posting the output right below the "<details>" tag, causing it to not render properly.  This is an effort to guide people away from doing that.